testing-common/src/main/java/io/opentelemetry/instrumentation/testing/junit/db/DbClientMetricsTestUtil.java

```diff
@@ -16,6 +16,7 @@ public class DbClientMetricsTestUtil {
 
   private DbClientMetricsTestUtil() {}
 
+  @SuppressWarnings("deprecation") // using deprecated db semconv
```

Comment:
imo this is not the correct way to handle this

Reply:
Nice! I've updated this PR with the content of https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/14015, there are a few extra things that aren't yet part of #14015 so it's still worth reviewing and merging it (the target branch is not `main` but #14015)